### DISCUSSION TITLE
need help 【 index::updater error】 求助

### DISCUSSION THREAD

**Discussion by jingmaster** [ordinals/ord#1870] [General]
我同步了全节点在安装ORD同步区块的时候出现index::updater error  已经勾选启用RPC且对区块没有裁剪。
![image](https://user-images.githubusercontent.com/58926137/222922619-1176c748-eef9-4941-abba-78cbcf61d87d.png)

我尝试修改bitcoin.conf文件，删除掉 txindex=1 前面的# 。再次同步，出现同样的错误。
尝试在bitcoin core中设置 【打开配置文件】记事本打开加入 txindex=1，再次同步，出现同样的错误。

求大佬帮帮忙找下问题出在哪里

**Comment by save19** [comment]
同样遇到这个问题，兄弟，你解决了吗

**Reply by jingmaster** [reply]
嗯，去bitcoin core的控制台把txindex同步到同一高度
![image](https://user-images.githubusercontent.com/58926137/223202562-1d58b203-4995-42f5-9e3e-0c4264c87f28.png)

**Reply by save19** [reply]
感谢，顶

**Comment by fatcatty** [comment]
![image](https://user-images.githubusercontent.com/83443605/234581082-cb1d5fe9-fe0c-45cd-8e32-97cc6ae43338.png)
兄弟这个提示false怎么办啊
